reV/batch/batch.py

```diff
@@ -295,15 +295,41 @@ def sub_dirs(self):
                     for tag in self.job_tags]
         return sub_dirs
 
+    @staticmethod
+    def _clean_arg(arg):
+        """Perform any cleaning steps required before writing an arg to a json
+
+        Cleaning steps:
+            1. Convert stringified dictionary to python dictionary object
+
+        Parameters
+        ----------
+        arg : str | int | float | dict
+            Value to be written to a batch job json file
+
+        Returns
+        -------
+        arg : str | int | float | dict
+            Cleaned value ready to be written to a batch job json file.
+        """
+
+        if isinstance(arg, str):
+            if '{' in arg and '}' in arg:
```

Comment:
What does the {/} have to do with replacing ' for "?

Reply:
the braces look for a dictionary that has been stringified. The ' vs " replacement has to do with how json wants strings with the double quotes.